### DISCUSSION TITLE
Identify root cause of build-errors.

### DISCUSSION THREAD

**Discussion by c2tarun** [Homebrew/discussions#1014] [Everyday usage] [answered]
Hi All,

We use homebrew's Analytics Formula build error metrics to track build-errors for `aws-sam-cli`. We saw a spike in errors in last few days are are trying to investigate the issue. However, I am not able to find any information about:
1. What is the build-error?
2. What version of homebrew is experiencing issue with our tool?
3. Pretty much any log related to that.
I wanted to ask is there any way I can find more information about those build errors from homebrew analytics and find the actual issue with our cli?

Thanks

**Comment by SMillerDev** [comment, marked as answer]
> What is the build-error?

You'll have to ask the users that got the error

> What version of homebrew is experiencing issue with our tool?

Homebrew usually auto updates so you can assume it's the latest version.

> Pretty much any log related to that.
> I wanted to ask is there any way I can find more information about those build errors from homebrew analytics and find the actual issue with our cli?

Homebrew analytics don't record that: https://docs.brew.sh/Analytics#what
